.eslintrc.js

```diff
@@ -25,76 +25,82 @@ module.exports = {
     'plugin:react/recommended',
   ],
 
-  plugins: [
-    'babel',
-    'react',
-    'json',
-    'import',
-  ],
+  plugins: ['babel', 'react', 'json', 'import', 'prettier'],
 
   globals: {
-    '$': 'readonly',
+    $: 'readonly',
     document: 'readonly',
     QUnit: 'readonly',
     window: 'readonly',
   },
 
   rules: {
+    'prettier/prettier': 'error',
+    'space-before-function-paren': [
```

Comment:
prettier is opinionated about this and does not support space before, except for anonymous functions which is *forces*